src/pages/reference/handlers/soap.md

```diff
@@ -0,0 +1,90 @@
+---
+title: SOAP handler | API Mesh for Adobe Developer App Builder
+---
+
+# SOAP
+
+<InlineAlert variant="warning" slots="text"/>
+
+The SOAP handler is experimental and should not be used in production deployments.
+
+The SOAP handler allows you to consume [SOAP](https://soapui.org) `WSDL` files and generate a remote executable schema for those services.
+
+Add SOAP handlers to your mesh:
+
+```json
+{
+  "meshConfig": {
+    "sources": [
+      {
+        "name": "SoapSource",
+        "handler": {
+          "soap": {
+            "source": "http://<Commerce Host>/soap?wsdl&services=customerCustomerRepositoryV1",
+            "wsdl": "http://<Commerce Host>/soap?wsdl&services=customerCustomerRepositoryV1",
+            "operationHeaders": {
+              "x-operation-header-key": "sample-x-operation-header-value"
+            },
+            "schemaHeaders": {
+              "x-schema-header-key": "sample-x-schema-header-value"
+            }
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+## SOAP limitations
+
+The experimental SOAP handler has the following limitations:
+
+- A `source` field and a `wsdl` field (with identical values) are required
+- `context.headers` are not available inside `operationHeaders`
+- `mutations` must be used when querying
```

Comment:
I believe one additional limitation of the soap handler (as mentioned by Narendra), is that the `httpdetails` plugin does not work (and thus would impact the [responseConfig.includeHTTPDetails](https://adobedocs.github.io/graphql-mesh-gateway/gateway/work-with-mesh/#include-httpdetails-in-query-responses) config. 

Would recommend confirming.